c2corg_ui/static/js/utils.js

```diff
@@ -0,0 +1,58 @@
+goog.provide('app.utils');
+
+goog.require('goog.asserts');
+
+
+/**
+ * @param {string} module The document type.
+ * @param {string} documentId The document id.
+ * @param {string} culture Culture.
+ * @return {string} Url.
+ */
+app.utils.buildDocumentUrl = function(module, documentId, culture) {
```

Comment:
> Do you mean the model of MVC?

No, I mean "module" (like "waypoints", "routes"). But that's maybe only a name used in Symfony (v5 framework).